graphenv/examples/hallway/hallway_state.py

```diff
@@ -64,7 +64,7 @@ def reward(self) -> float:
             float: random reward between 0 and 2 on the goal vertex, -0.1
                 otherwise.
         """
-        return random.random() * 2 if self.cur_pos >= self.end_pos else -0.1
+        return random.random() * 2 if self.cur_pos >= self.end_pos - 1 else -0.1
```

Comment:
weren't we planning to set `self.end_pos = corridor_length - 1`? That would probably be a simpler edit, although some of the tests might still need to get changed

Reply:
i like changing the end_pos definition, seems like a cleaner way to go -- pushed that change just now!